src/main/java/com/isa/morswiny/eventsDao/EventCRUDRepositoryInterface.java

```diff
@@ -1,4 +1,11 @@
 package com.isa.morswiny.eventsDao;
 
+import com.isa.morswiny.events.Event;
+
+
+
+
 public interface EventCRUDRepositoryInterface {
+
+    Event getEventByID(Integer id);
```

Comment:
Metoda getEventByID nie powinna byc w EventSearchRepositoryInterface?

Reply:
Nie wiem, a to crud to właśnie nie jest do odczytywania, tworzenia, usuwania i modyfikowania???

Reply:
CRUD jasne, tak jak mowisz :) metoda skojarzyla mi sie z wyszukiwaniem, do ktorego mamy interfejs i klase, ale mysle ze tu jako read tez jest ok.